src/common/config/defaultPreferences.ts

```diff
@@ -31,7 +31,7 @@ const defaultPreferences: ConfigV3 = {
     useSpellChecker: true,
     enableHardwareAcceleration: true,
     autostart: true,
-    spellCheckerLocale: 'en-US',
+    spellCheckerLocales: [],
```

Comment:
should we default to english again?

Reply:
The app already defaults to whatever language your system, so this is unnecessary.